build_openroad.sh

```diff
@@ -44,37 +55,29 @@ while (( "$#" )); do
       OR_REPO=$2
       shift 2
       ;;
-    -d|--dev_repo)
-      UPDATE_PRIVATE=1
-      shift
-      ;;
     -l|--latest)
       UPDATE_OR=1
       shift
       ;;
+    -t|--threads)
+      PROC=$2
+      shift 2
+      ;;
     -n|--nice)
       NICE="nice"
-      PROC=`nproc --all`
+      PROC=$(nproc --all)
       shift
       ;;
     -o|--local)
       BUILD_METHOD="LOCAL"
       shift
       ;;
-    -p|--platform)
-      UPDATE_PLATFORM=1
-      shift
-      ;;
-    --no_init)
-      OPENROAD_FLOW_NO_INIT=1
-      shift
-      ;;
```

Comment:
Why remove these? I'd at least like to keep `--no_init` so that I can rebuild OR without having git complain about untracked files. I can understand removing platforms as we no longer keep private platforms inside `flow/platforms` but then you might as well remove the code for updating as well.

Reply:
These options are not used anywhere (at least according to running `grep -r` on the master branch):

```
grep -r OPENROAD_FLOW_NO_INIT
build_openroad.sh:      OPENROAD_FLOW_NO_INIT=1     -> this is line 69 that I am proposing removing
```

```
grep -r UPDATE_PLATFORM
build_openroad.sh:      UPDATE_PLATFORM=1      -> line 65, proposing to remove
build_openroad.sh:if [ ! -z ${UPDATE_PLATFORM+x} ]; then   -> 135 to 145, proposing to remove
```

If they are used in an external script (although improbable since they are not exported) let me know and I will keep.

Reply:
Oh that's because line 69 is supposed to be `OPENROAD_FLOW_NO_GIT_INIT=1` to correspond with line 100 https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/f6fdc33bdae9f3213621c8504c5ba739fd8ab60d/build_openroad.sh#L100

I thought that was fixed in a PR but maybe not?

Reply:
Fixed on 1af0dd0

Reply:
--local is useful to people who want to build OR without docker afaik.  Do we mention it in the docs?

Reply:
yes, on the current [docs here](https://openroad.readthedocs.io/en/latest/user/GettingStarted.html#option-2-build-from-sources-locally), and on the [new here](https://openroad-test.readthedocs.io/en/latest/user/BuildLocally.html#clone-and-build)